docs/section11/Rule11-16.md

```diff
@@ -0,0 +1,39 @@
+# ServiceWaterHeating - Rule 11-16
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-16  
+
+**Rule Description:** "Gas water heaters shall be modeled using natural gas as their fuel.  Exceptions: Where natural gas is not available for the proposed building site, as determined by the rating authority, gas water heaters shall be modeled using propane as their fuel."  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (h)  
+
+**Evaluation Context:** Each SWH Equipment  
+**Data Lookup:**   
+**Function Call:**
+
+
+**Applicability Checks:**
+- look at each SHW Equipment: `for swh_eq in B_RMD.service_water_heating_equipment:`
+  - if the fuel source is electricity, the rule is not applicable: `if shw_eq.heater_fuel_type == "ELECTRICITY": NOT_APPLICABLE`
+  - otherwise, continue to rule logic: `CONTINUE TO RULE LOGIC`
+  
+    ## Rule Logic:
+    - create variable heater_fuel_type: `heater_fuel_type = swh_eq.heater_fuel_type`
+    - create a list that will be used to store the fuel types used in the proposed model swh equipment: `proposed_swh_eq_fuel_types = []`
+    - look at each swh equipment in the proposed model and collect a list of fuel types: `for swh_eq_p in P_RMD.service_water_heating_equipment: if !(swh_eq_p.fuel_type.in? proposed_swh_eq_fuel_types): proposed_swh_eq_fuel_types.append(swh_eq_p.fuel_type)`
```

Comment:
I recommend that the RDS look at all equipment. If gas is used for space heating equipment or miscellaneous equipment then that would tell us it is available on site.

Reply:
Also, looks like some Ruby syntax was used.